src/libexpr/get-drvs.cc

```diff
@@ -102,6 +102,24 @@ StorePath DrvInfo::queryOutPath() const
 }
 
 
+std::vector<std::string> DrvInfo::queryOutputNames()
+{
+    std::vector<std::string> outputNames;
+
+    Bindings::iterator i;
+    if (attrs && (i = attrs->find(state->sOutputs)) != attrs->end()) {
+        state->forceList(*i->value, *i->pos);
+        for (auto elem : i->value->listItems()) {
+            std::string name(state->forceStringNoCtx(*elem, *i->pos));
+            outputNames.push_back(name);
+        }
+    } else
+        outputNames.push_back("out");
```

Comment:
curly brace here my preference, maybe with a comment explaining why

Reply:
This is also documented

https://github.com/NixOS/nix/blob/d5322698a2abbc6d141e1d244e17b0d226a2f18b/doc/manual/src/expressions/derivations.md?plain=1#L50-L51

and it's the same logic as in `queryOutputs` so I'd rather not add a comment.

Reply:
There's also no braces in `queryOutputs` so leaving it alone for symmetry.